sycl/doc/extensions/experimental/sycl_ext_oneapi_kernel_properties.asciidoc

```diff
@@ -60,21 +41,27 @@ the following extensions:
 
 - link:sycl_ext_oneapi_properties.asciidoc[sycl_ext_oneapi_properties]
 
-== Feature Test Macro
 
-This extension provides a feature-test macro as described in the core SYCL
-specification section 6.3.3 "Feature test macros".  Therefore, an
-implementation supporting this extension must predefine the macro
-`SYCL_EXT_ONEAPI_KERNEL_PROPERTIES` to one of the values defined in the table
-below.  Applications can test for the existence of this macro to determine if
-the implementation supports this feature, or applications can test the macro's
-value to determine which of the extension's APIs the implementation supports.
+== Status
 
-[%header,cols="1,5"]
-|===
-|Value |Description
-|1     |Initial extension version.  Base features are supported.
-|===
+This is an experimental extension specification, intended to provide early
+access to features and gather community feedback.  Interfaces defined in this
+specification are implemented in {dpcpp}, but they are not finalized and may
+change incompatibly in future versions of {dpcpp} without prior notice.
+*Shipping software products should not rely on APIs defined in this
+specification.*
+
+== Version
+
+Revision: 1
```

Comment:
The template does not  have a "Version" section because it seemed confusing to have both a "Version" for the document and a feature-test macro version.  Did you add this because the feature-test macro says "The APIs of this experimental extension are not versioned, so the feature-test macro always has this value"?

Reply:
I kept it here to preserve the revision history at the end of the document. Would you prefer we drop it?

Reply:
Yes.  We dropped it from the template because people kept forgetting to update it, and no one had a clear understanding of when the document version should be updated.

Reply:
"Version" and "Revision History" sections have been removed.